README.md

```diff
@@ -15,3 +15,14 @@ GET /wishlists | LIST | Show all wishlists
 GET /wishlists?q=querytext | QUERY | Search for a wishlist
 GET /wishlists/`<id>`?q=querytext | QUERY | Search for items in wishlist
 PUT /wishlists/`<id>`/items/`<itemid>`/add-to-cart | ACTION | Move an item from wishlist to the shopping cart
+
+To execute the service run:
+
+```
+vagrant up --provision
+vagrant ssh
+cd /service
+FLASK_APP=service:app flask run -h 0.0.0.0
+```
+
+Then open a browser session to http://0.0.0.0:5000/
```

Comment:
I'm not sure if `0.0.0.0` is correct. I think it should be `localhost` or `127.0.0.1`. It doesn't work on Windows. Not sure about Mac/Linux.

Reply:
That is how It works in my Mac, how does it work for you?

Reply:
Going to `http://0.0.0.0:5000/` says `This site can’t be reached`. I need to put `http://localhost:5000/` or `http://127.0.0.1:5000/` to get to the service. Probably this is just a Windows thing

Reply:
localhost also works on my Mac, so I'll leave it with localhost.